phone-book/src/contacts.js

```diff
@@ -1,39 +1,73 @@
-
-
 class Contacts {
-  constructor(containerSelector) {
-    this.container = document.body.querySelector(containerSelector);
+  constructor(app) {
+    this.title = "Contacts";
+    this.app = app;
+    this.appContainer = app.appContainer;
     this.pageName = 'Contacts';
-    this.users = users;
-    this.usersOrderDirection = 'asc';
-    this.container.addEventListener('click', (event) => {
+    this.users = [];
+    this.usersOrderDirection = 'asc';    
+  }
+
+  initEvents(){
+    this.appContainer.querySelector("main > div.container").addEventListener('click', (event) => {
```

Comment:
you could add your even to window or document for example, but it's a bad practice. much better to make such area smaller and make it relate to part on UI